services/cases-api/helpers/dynamo.js

```diff
@@ -0,0 +1,19 @@
+import dynamoDb from '../../../libs';
+import config from '../../../config';
+
+export async function getCaseWhereUserIsApplicant(caseId, userPersonalNumber) {
+  const PK = `USER#${userPersonalNumber}`;
+  const SK = `${PK}#CASE#${caseId}`;
+
+  const dynamoDbQueryParams = {
+    TableName: config.cases.tableName,
+    ExpressionAttributeValues: {
+      ':pk': PK,
+      ':sk': SK,
+    },
+    KeyConditionExpression: 'PK = :pk AND SK = :sk',
+    Limit: 1,
+  };
+
+  return dynamoDb.call('query', dynamoDbQueryParams);
+}
```

Comment:
Will this support the new GSI implementation on cases db?

Reply:
There should not be any interference with the new GSI implementation